feat(content): Give korath worldships names instead of serial numbers (#9696)
endless-sky/endless-sky: data/korath/korath ships.txt

	explode "large explosion" 40
	explode "huge explosion" 50
	"final explode" "final explosion large"

> 	description "The largest of Korath production vessels are derived from a common and ancient chassis. Karek A'awoj are adapted to haul materials between refinery and factory stations. They're named after a productive colony of the Efreti, to honor their focus on peaceful industry."

Reviewer: ```suggestion
	description "The largest of Korath production vessels are derived from a common and ancient chassis. The simplest are adapted to haul materials between refinery and factory stations. Fittingly, they are named after a productive colony of the Efreti, to honor their focus on peaceful industry."
```

Each of the five derivative descriptions has a physical descriptor of some kind, which went missing here since I last reviewed.  Also, starting a sentence with "They're" felt far too... conversational?  Probably the biggest disagreement UnorderedSigh and I have is over how simple the English should be.

	explode "large explosion" 40
	explode "huge explosion" 50
	"final explode" "final explosion large"

> 	description "The largest of Korath production vessels are derived from a common and ancient chassis. This deceptively spindly ship bristles with weapon emplacements for heavy raiding or defending other vessels. Its name is in memory of the Mereti, who made powerful warships."

Reviewer: Fleshing this out a bit...

```suggestion
	description "The largest of Korath production vessels are derived from a common and ancient chassis. This deceptively spindly ship bristles with weapon emplacements for heavy raiding or defending other vessels. It is named after the Mereti's capital planet, where they made powerful warships out of modular components."
```

	explode "large explosion" 40
	explode "huge explosion" 50
	"final explode" "final explosion large"

> 	description "The largest of Korath production vessels are derived from a common and ancient chassis. This relatively streamlined ship has been adapted to collect, refine, and transport hyperspace fuel. Its name refers to the delicious squids that lived in gas giants like Deka Dathnak."

Reviewer: ```suggestion
	description "The largest of Korath production vessels are derived from a common and ancient chassis. This relatively streamlined ship has been adapted to collect, refine, and transport hyperspace fuel. Its name refers to the delicious squids that lived in their gas giants, especially Deka Dathnak."
```

Wanted to restore the possessive here, but it broke the flow, hence the addition of especially.

Author: That still feels a bit awkward.

> 	description "The largest of Korath production vessels are derived from a common and ancient chassis. This relatively streamlined ship has been adapted to collect, refine, and transport hyperspace fuel. Its name is in memory of the delicious squids that lived in gas giants of their home sector, such as Deka Dathnak."

Author: I just checked and Deka Dathnak is on the other side of Korath space, so their home sector would probably just be Korath space in general.
![image](https://github.com/endless-sky/endless-sky/assets/109186806/254b16de-fa55-432b-8b62-5aa1bab2f38b)

> 	description "The largest of Korath production vessels are derived from a common and ancient chassis. This relatively streamlined ship has been adapted to collect, refine, and transport hyperspace fuel. Its name is in memory of the squids that lived in the Korath gas giants back in their home sector, which were a delicacy."

Reviewer: I think this word order flows better..

```suggestion
	description "The largest of Korath production vessels are derived from a common and ancient chassis. This relatively streamlined ship has been adapted to collect, refine, and transport hyperspace fuel. Its name is in memory of the delicious squids that lived in the gas giants of their home sector, such as as Deka Dathnak."
```

	explode "large explosion" 40
	explode "huge explosion" 50
	"final explode" "final explosion large"

> 	description "The largest of Korath production vessels are derived from a common and ancient chassis. This many-lobed ship has been adapted to carry fighters into large-scale raids, or mining craft into asteroid belts. Its radial arrangements of hangars allows fighters a maximum variety of safe approach patterns in combat. Its name is in memory of the Sestor, who made carriers and fighters."

Reviewer: Just changing up the pattern a bit so these aren't repetitive.

```suggestion
	description "The largest of Korath production vessels are derived from a common and ancient chassis. This many-lobed ship has been adapted to carry fighters into large-scale raids, or mining craft into asteroid belts. Its radial arrangements of hangars allows fighters a maximum variety of safe approach patterns in combat. Sestor exiles, who had a great affinity for carriers and fighters, bequeathed this variant its name after their capital world."
```

	explode "large explosion" 40
	explode "huge explosion" 50
	"final explode" "final explosion large"

> 	description `Originally designed as a luxury habitat for those wealthy enough to escape living on the polluted surface of Korath worlds, when the Korath were exiled, it was into these gargantuan "world-ships" they first crammed themselves into and fled to the core on. Nostalgically, the class was then renamed after the devastated and now forbidden Korath homeworld, in memory of the Korsmanathi, ancestors of all Korath.`

Reviewer: Comma seems inappropriate and memory is redundant with nostalgia

```suggestion
	description `Originally designed as a luxury habitat for those wealthy enough to escape living on the polluted surface of Korath worlds, when the Korath were exiled, it was into these gargantuan "world-ships" they first crammed themselves into and fled to the core on. Nostalgically, the class was then renamed after the devastated and now forbidden Korath homeworld, in memory of the Korsmanathi, ancestors of all Korath.`
```
```suggestion
	description `Originally designed as a luxury habitat for those wealthy enough to escape living on the polluted surface of Korath worlds, when the Korath were exiled, it was into these gargantuan "world-ships" they first crammed themselves into and fled to the core on. Nostalgically, the class was then renamed after the devastated and now forbidden Korath homeworld in honor of the Korsmanathi, ancestors of all Korath.`
```
